src/core/api/abuse.js

```diff
@@ -20,3 +19,38 @@ export function reportAddon(
     state: api,
   });
 }
+
+export type ReportUserParams = {|
+  api: ApiStateType,
+  message: string,
+  user: UserType,
+|};
+
+export type ReportUserResponse = {|
+  message: string,
+  reporter: {|
+    id: number,
+    name: string,
+    url: string,
+    username: string,
+  |} | null,
+  user: {|
+    id: number,
+    name: string,
+    url: string,
+    username: string,
+  |},
+|};
+
+export function reportUser(
+  { api, message, user }: ReportUserParams
```

Comment:
Ah, shoot. Right. Testing this means rebasing the user profile patch onto it which I've been trying to avoid doing, and the tests didn't catch this. Thanks.